checkpoints/check_audio.py

```diff
@@ -23,10 +22,12 @@
 import textwrap
 import time
 import traceback
+try:
```

Comment:
Please restore the original behaviour here. If you have a reason to make this change, do it in a separate pull request.